Configuration.cs

```diff
@@ -0,0 +1,23 @@
+using Microsoft.Extensions.Configuration;
+class JobsConfig
+{
+    public string JobsDirectory{get; private set;}
+    public string JobsPrefix{get; private set;}
+    public string JobsURL{get; private set;}
+    private static JobsConfig instance;
+    private JobsConfig(string file_path)
+    {
+        var config=new ConfigurationBuilder().AddJsonFile(file_path).Build();
```

Comment:
We don't have to pay for the whitespace used in the code. There's no need to cram everything together. :)

Spaces between operators, newlines separating things things that start new scopes or that is a different 'section' of stuff being done,  etc.